### DISCUSSION TITLE
add chat bot

### DISCUSSION THREAD

**Discussion by codetil** [spaciousejar/learn-git-with-me#3] [Q&A]
wants a chatbot

**Comment by spaciousejar** [comment]
> wants a chatbot



Nice idea! 🤖

**Comment by Anit-K-Peter** [comment]
Hey there! If you're looking for a chatbot solution, you can try my bot code from this repository: [Simple Gemini ChatBot](https://github.com/Anit-K-Peter/Simple-Gemini-ChatBot).

**Reply by spaciousejar** [reply]
Hey, thanks for the suggestion! I'll check out the Simple Gemini ChatBot repository for a chatbot solution.
